lib/rucio/tests/test_rule.py

```diff
@@ -88,6 +88,17 @@ def check_dataset_ok_callback(scope, name, rse, rule_id, session=None):
     return False
 
 
+@transactional_session
+def check_rule_progress_callback(scope, name, progress, rule_id, session=None):
+    callbacks = session.query(models.Message.id).filter(models.Message.payload == json.dumps({'scope': scope,
```

Comment:
there's no index on the message payload column. is this ok? in case we have a backlog of messages, wouldn't this slowdown?

Reply:
Yes, this method is just used for testing.